test/integration/functional_test.go

```diff
@@ -72,9 +73,25 @@ var runCorpProxy = detect.GithubActionRunner() && runtime.GOOS == "linux" && !ar
 
 // TestFunctional are functionality tests which can safely share a profile in parallel
 func TestFunctional(t *testing.T) {
+	testFunctional(t, "")
+}
+
+// TestFunctionalNewestKubernetes are functionality run functional tests using NewestKubernetesVersion
+func TestFunctionalNewestKubernetes(t *testing.T) {
```

Comment:
if the default version and the newest version are exactly the same, we should Skip the newest test with a good message that it is gonna be duplicate